js/sparkswap.js

```diff
@@ -200,8 +210,22 @@ module.exports = class sparkswap extends Exchange {
         throw new ExchangeError ('Not Implemented');
     }
 
-    async commit () {
-        throw new ExchangeError ('Not Implemented');
+    async commit (symbol = '', balance = '', market = '', params = {}) {
```

Comment:
I think we need another name for symbol because ccxt symbol === marketName. Maybe currency?

Reply:
@martineehrlich good catch. In their examples they use `code`.